charms/worker/k8s/src/config/extra_args.py

```diff
@@ -18,13 +19,18 @@
 )
 
 
-def _parse(config_data) -> Dict[str, str]:
+def _parse(option: StrOption, charm: ops.CharmBase) -> Dict[str, str]:
```

Comment:
Can we target the super class `CharmOption`?

Reply:
um, if we do, the type of the value in `option.get(charm)` wouldn't be `str` -- it'd be an `Any`.